conformance/build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
 }
 
 group 'org.jspecify'
-version '0.1'
+version '0-SNAPSHOT'
```

Comment:
I went with full-on "0.0.0-SNAPSHOT" for Semantic Versioning reasons, but I am not overly invested in that.

Reply:
Being consistent between this and jspecify/jspecify would be nice.

Reply:
OK.